.github/workflows/template-ui-tests.yml

```diff
@@ -42,13 +49,46 @@ jobs:
           retry-delay: 10s
 
       - name: Run Playwright tests
-        run: npx playwright test ${{ inputs.tests_to_run }}
+        id: run
+        shell: pwsh
+        run: |
+          # On Success - 'Running x tests using 1 worker ···· x passed'
+          $testOutput = npx playwright test ${{ inputs.tests_to_run }}
+
+          $testOutputString = $testOutput.ToString()
+
+          # Check if tests passed (i.e., no failures detected)
+          $testPassed = ($testOutputString -match "failed") -eq $false # On Failure - 'Running x tests using 1 worker ···· x passed, x failed'
+
+          Write-Host "Test ouput: $testOutput"
+          Write-Host "Test passed: $testPassed"
+
+          echo "testPassed=$testPassed" >> $env:GITHUB_OUTPUT
         env:
           HOST_URL: ${{ inputs.deploy_url }}
 
       - uses: actions/upload-artifact@v4
         if: always()
+        id: artifact-report
         with:
           name: playwright-report
           path: playwright-report/
           retention-days: 2
+
+  create-an-issue:
+    name: GitHub Issue
+    runs-on: ubuntu-latest
+    needs: test
+    if: needs.test.outputs.testPassed == 'False' && inputs.create-issue == true
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Create an Issue
+        uses: JasonEtco/create-an-issue@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          TITLE: "Broken Images on Homepage"
+          ARTIFACT_ID: ${{ needs.test.outputs.artifact-id }}
+          GITHUB_RUN_ID: ${{ github.run_id }}
+        with:
+          filename: .github/ISSUE_TEMPLATE/auto_bug_report.md
```

Comment:
Personally, I would've included this in the `daily-image-tests` workflow

Reply:
done - removed it from the flow and now returning the outcome to the first flow